tests/integrations/config/session.py

```diff
@@ -5,11 +5,8 @@
     "default": "cookie",
     "cookie": {},
     "redis": {
-        "driver_config": {
-            "host": "127.0.0.1",
-            "port": "6379",
-            "password": "",
-        },
+        "host": "127.0.0.1",
```

Comment:
I know you put default values inside Redis session driver connection, but could you in addition put "port" and "password" here again ? It will help users wanting to use this driver to configure it as they will directly see those base options. 👍 Also it will more consistent with options of redis cache driver.

Reply:
Yeah Makes sense...
Done